.gitignore

```diff
@@ -11,6 +11,7 @@ deployments/teku
 deployments/postgres-data
 deployments/metabase-data
 deployments/.env
+deployments/oracle-data/oracleLogs.txt
```

Comment:
can you add this to the gitignore? this file shouldnt be part of the repo

Reply:
It already is! This file is .gitignore